develop/plone/forms/wysiwyg.rst

```diff
@@ -30,15 +30,24 @@ If you need to put a `<script>` tag on your content text in TinyMCE you can disa
 
         If you don't trust all of your site editors, then this will open your site for an attack.
 
-**Step 1:** Turn off Plone’s safe_html transform. Go to /portal_transforms/safe_html in the Management Interface, and enter a 1 in the ‘disable_transform’ box. This prevents Plone from removing tags and attributes while rendering rich text.
+Step 1: Turn off Plone’s safe_html transform
+--------------------------------------------
 
-**Step 2:** Set the "X-XSS-Protection: 0" response header. This can be done in your frontend webserver such as apache or nginx. Alternatively, if you only need to disable the protection for users who have permission to edit, you can add this to the site’s main_template:
+Go to the Site Setup, HTML Filtering settings (`@@filter-controlpanel`). Check the `Disable HTML filtering` box. This prevents Plone from removing tags and attributes while rendering rich text.
+
+Step 2: Set the "X-XSS-Protection: 0" response header
+-----------------------------------------------------
+
+This can be done in your frontend webserver such as apache or nginx. Alternatively, if you only need to disable the protection for users who have permission to edit, you can add this to the site’s main_template:
```

Comment:
```suggestion
This can be done in your frontend webserver such as Apache or Nginx. 
Alternatively, if you only need to disable the protection for users who have permission to edit, you can add this to the site’s main_template:
```